en_ID/main.css

```diff
@@ -1,99 +1,98 @@
-
 @import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
 
 * {
-    margin: 0;
```

Comment:
Please don't create unnecessary changes like removing space like this.

Reply:
Sorry, I won't repeat the same mistakes again.